### ISSUE TITLE
ERROR: Only releases with status draft may be created on draft app

### ISSUE THREAD

**Issue by zommerfelds** [eventOneHQ/apkup#49]
Hi! First of all, thank you very much for this tool.

I'm having an issue similar to https://github.com/eventOneHQ/apkup/issues/12.

I'm new to setting up apps in the Google Play Console. Right now the store listing and app config are not completed. However, I was able to manually create an "Internal testing" release yesterday:

![image](https://user-images.githubusercontent.com/1260622/111123687-f1d40480-856f-11eb-8038-ea0330f55fb0.png)

I'm running this inside Docker and getting the result below:

```
#28 [upload-to-play 4/5] RUN apkup --version
#28 sha256:016d1c8b83d9855cc000f0d5eb017be5e98db60aebbd19c14c31cb585c8f4e66
#28 1.418 1.3.1
#28 DONE 1.4s

#29 [upload-to-play 5/5] RUN --mount=type=secret,id=service-account apkup     --key /run/secrets/service-account     --apk app-release.apk     --track internal
#29 sha256:219678f208b48000268e3ce738aaa25f4175cc4be2580f8e67505087aa8c65a8
#29 1.610 - Uploading APK...
#29 5.519 ERROR: Only releases with status draft may be created on draft app.
```

Also, please note that I haven't managed to install the test release on my phone yet. I joined the testing program via the link but Google Play crashes when I click the link to Play... Maybe I've been inpatient and just need to wait. But the error above is wouldn't be helpful in this case.

Any comment would be helpful.

**Comment by zommerfelds** [comment]
So it seems like the Google Play API doesn't let you upload the release in the `COMPLETED` state when the app is still in draft mode.

I was able to hack the tool to do what I need.

https://github.com/eventOneHQ/apkup/compare/master...zommerfelds:master

Basically you need to create a release in 'DRAFT' mode and then promote it in the same track to 'COMPLETED'.

If you like this I could help introduce some sort of `--draft-mode` flag.

**Comment by zommerfelds** [comment]
Actually, my diff above is not sufficient, I'm still getting the error. There is probably more that needs to be tested. Maybe we can open a ticket with Google.

**Comment by zommerfelds** [comment]
I finished setting up my app from the Dashboard page (Set up your app). After this step the app is no longer in "Draft" state. It says "Closed testing" in the app selector.

![image](https://user-images.githubusercontent.com/1260622/111538048-81460700-876c-11eb-8099-cdc0b3aa11d2.png)

This resolved the error for me. In the end the error is pretty obvious, but it was unclear to me what parts of the forms I needed to submit.

I'm gonna close this since there is nothing to do really other than give Google feedback.
